### DISCUSSION TITLE
Issue regarding rule|text|BR-BT-01311-0150

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#642] [Q&A] [answered]
Hi,
During testing we got the error message for rule|text|BR-BT-01311-0150, meaning:
"Deadline for receipt of requests to participate' (BT-1311) must be the same as 'End date' (BT-537), for a dynamic purchasing system. 

The dates that where applied in the test where the follwing:
"PlannedPeriodEndDate": "2023-12-10T00:00:00+02:00" (BT-537 Duration end date)
"DeadlineReceiptRequests": "2023-12-10T09:43:00+02:00" (BT-1311 Deadline receipt requests)

The dates _are_ the same and yet we get this error. Is the rule not correctly implemented due to the fact that both date and time are compared even though "BT-537 Duration end date" does not require time? 

Kind regards,
Åsa

**Comment by YvesJo** [comment, marked as answer]
Hi,
Controls are applied on fields, not on Business Terms. In this case, the control checks that, when both exist, the content of both fields BT-1311(d)-Lot and BT-537-Lot do match.
Please note that both of these fields are of type Date and not DateTime and their content should be "2023-12-10+02:00" instead. For the deadline, the time is specified in another field 1311(t)-Lot associated to another XML element.

**Comment by rouschr** [comment]
To complement the answer of Yves, you can find field metadata in the SDK [fields/fields.json (here 1.8.0) file](https://github.com/OP-TED/eForms-SDK/blob/1.8.0/fields/fields.json):
```
 }, {
    "id" : "BT-537-Lot",
    "parentNodeId" : "ND-LotDuration",
    "name" : "Duration End Date",
    "btId" : "BT-537",
    "xpathAbsolute" : "/*/cac:ProcurementProjectLot[cbc:ID/@schemeName='Lot']/cac:ProcurementProject/cac:PlannedPeriod/cbc:EndDate",
    "xpathRelative" : "cbc:EndDate",
    "xsdSequenceOrder" : [ { "cbc:EndDate" : 4 } ],
    "type" : "date",
    "legalType" : "DATE", 
    ...
```
